# Configuration for mobile device has valid used mobile application functionalities

Checks that used mobile application functionalities of the configuration for mobile device is correct for current
runtime version.

## Noncompliant Code Example

## Compliant Solution

## See
