# Configuration has valid mobile application URLs

Checks for native configuration for mobile device since 8.3.18 version that for all mobile application URLs
the base URL in the link is specified

## Noncompliant Code Example

## Compliant Solution

## See
