# Configuration has valid compatibility mode version

Checks for native configuration that the current compatibility mode must be greater than or equal
to version 8.3.6 and not higher than the used runtime version

## Noncompliant Code Example

## Compliant Solution

## See
