# Metadata referential integrity check

Checks the referential integrity of metadata objects in a configuration. Any lost reference which potentially prevents the configuration from the deployment and/or normal work in 1C:Enterprise is a subject for this check

## Noncompliant Code Example

## Compliant Solution

## See

