# Проверка целостности ссылок объектов метаданных

Проверяет ссылочную целостность объектов метаданных в конфигурации. Любая потерянная ссылка потенциально способна помешать публикации конфигурации в информационную базу и/или нормальной работе 1С:Предприятие

## Noncompliant Code Example

## Compliant Solution

## See

